modules/internal/Atomics.chpl

```diff
@@ -467,15 +467,15 @@ module Atomics {
       return ret;
     }
     inline proc add(value:uint(8), order:memory_order = memory_order_seq_cst):void {
-      on this do atomic_fetch_add_explicit_uint_least8_t(_v, value, order);
+      on this do local atomic_fetch_add_explicit_uint_least8_t(_v, value, order);
```

Comment:
Ok, from one of your comments it sounded like you thought you were adding local-ons, so I wanted to make sure you knew this was a local-block.

Still we're trying to remove local blocks (and deprecate the feature) so it seems unfortunate to be adding more of them. In this case it seems strange that the compiler would ever generate a get for an extern proc.

pinging @benharsh, our wide pointer expert